server/models/User.js

```diff
@@ -0,0 +1,57 @@
+const mongoose = require('mongoose')
+const UserSchema = new mongoose.Schema({
+    firstName: {
```

Comment:
I think It's better to write just "full_name" instead of separated. It's easier to filter on just one field.